contrib/packs/actions/restart_component.yaml

```diff
@@ -10,7 +10,7 @@
       immutable: true
     cmd:
       immutable: true
-      default: "st2ctl restart-component {{servicename}}"
+      default: "service {{servicename}} restart"
```

Comment:
I would like to ask only one thing, is `service {{svc}} restart` portable between all unix OSes?
fedora, debian, redhat etc all support it?